libs/SalesforceSDK/src/com/salesforce/androidsdk/auth/AuthenticationUtilities.kt

```diff
@@ -144,68 +202,20 @@ internal suspend fun onAuthFlowComplete(
         .clientId(consumerKey)
         .nativeLogin(nativeLogin)
         .build()
-    account.downloadProfilePhoto()
```

Comment:
Moved this one into `fun addAccount(account: UserAccount?, context: Context, isTestRun: Boolean, loginServerManager: LoginServerManager)`